### PR TITLE
Fixed navbar and also different branch

### PR DESCRIPTION
**Pull request by YadlaMani**
### Issue Title: Adding a footer to the website

- **Name of the project** : 
- **Category:** 
- **Name:** 
- **GitHub ID:** 
- **Idenitfy yourself:** 

Closes:#142

### Describe the add-ons or changes you've made 📃

Give a clear description of what have you added or modifications made
Added complete footer section
included social links,and repo links

### Type of change ☑️

What sort of change have you made:

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [X] Code style update (formatting, local variables)
- [X] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update

### Checklist: ☑️

- [X] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [X] This PR does not contain plagiarized content.
- [X] I have performed a self-review of my own code.
- [X] I have commented my code, particularly wherever it was hard to understand.
- [X] My changes generate no new warnings.

### Screenshots 📷

image

### CODE REVIEW

**Comment by YadlaMani** [comment]
@Avdhesh-Varshney pls review this

File: src/App.jsx
Comment (Avdhesh-Varshney) [inline src/App.jsx:1]: A lots of un-necessary changes not in this file but many other files.
Keep your tab size by default 4 not 2.

File: src/components/Pages/MainPage.jsx
Comment (Avdhesh-Varshney) [inline src/components/Pages/MainPage.jsx:1]: Remove these un-necessary changes done by you.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
@YadlaMani These extra spaces removes and converting single quote into double quote is not required. Just add your code only.

**Comment by YadlaMani** [comment]
@Avdhesh-Varshney but it's the general convection of writing standard code right

**Pull request closed without merging**